src/include/platform/CHIPDeviceConfig.h

```diff
@@ -826,6 +826,18 @@
 #define CHIP_DEVICE_CONFIG_DEFAULT_TELEMETRY_INTERVAL_MS 90000
 #endif
 
+/**
+ * @def CHIP_DEVICE_CONFIG_ETHERNET_IF_NAME
+ *
+ * @brief
+ *   Default ethernet network interface name used to centralize all metrics that are
+ *   relevant to a potential Ethernet connection to a Node.
+ *
+ */
+#ifndef CHIP_DEVICE_CONFIG_ETHERNET_IF_NAME
+#define CHIP_DEVICE_CONFIG_ETHERNET_IF_NAME "enp0s25"
```

Comment:
How can we possibly have any sane default interface name in this shared config file?  Isn't this by definition platform-specific?

Reply:
This is the default config value for platform layer, each platform can overwrite this config within their own CHIPDeviceBuildConfig.h

Reply:
Maybe we can remove the generic reference name and move its define to CHIPDeviceBuildConfig.h directly.

Reply:
Addressed in https://github.com/project-chip/connectedhomeip/pull/9647